### DISCUSSION TITLE
How to Enable this plugin in the End

### DISCUSSION THREAD

**Discussion by ZombieGamerz-G** [GriefPrevention/GriefPrevention#1516] [Q&A]
If I try to use the Plugin in the End it says "Land Claims are disabled in this world" how do I fix that?

**Comment by Jikoo** [comment]
[Configure the worlds GP runs in](https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#limiting-which-worlds-griefprevention-runs-on).

**Reply by ZombieGamerz-G** [reply]
I do not understand can you be more detailed

**Reply by bigpresh** [reply]
The link provided is to the documentation on exactly how to configure which worlds GriefPrevention is active in - did you read it?  Suggestions on ways to make it clearer would be welcomed if it wasn't enough.

If you look at your GP config file - `plugins/GriefPreventionData/config.yml` you'll see something like the following:

```
  Claims:
    Mode:
      world: Survival
      world_nether: Disabled
      world_the_end: Disabled
```

If you want it to be active in the end, you'd change the "Disabled" next to `world_the_end` to `Survival`, `Creative` or `CreativeRequiringClaims` depending on how you would like GP to act.

Does that make it clearer?

(After updating the config, you'll want to run `/gpreload` to make it take effect - or edit the config with the server stopped, then start it back up afterwards)

**Reply by ZombieGamerz-G** [reply]
Thanks! I appreciate it!
